Packs/RedCanary/Integrations/RedCanary/RedCanary.py

```diff
@@ -590,7 +600,9 @@ def main():
         command_func = COMMANDS.get(demisto.command())
         if command_func is not None:
             if demisto.command() == 'fetch-incidents':
-                demisto.incidents(fetch_incidents())
+                last_run, incidents = fetch_incidents()
+                demisto.incidents(incidents)
+                demisto.setLastRun(last_run)
```

Comment:
if we are taking out `setLastRun` it makes sense to do so for `getLastRun` and pass it to `fetch_incidents`.
or to put them all-together inside. but not to mix.

Reply:
@yuvalbenshalom we do check `list_detections` in Test which is the function we use in fetch.